### PR TITLE
Release 2.0.1

### PR DESCRIPTION
**Pull request by PayamEmami**
This is a patch release addressing a few bugs

https://github.com/nf-core/metaboigniter/issues/86
https://github.com/nf-core/metaboigniter/issues/83
https://github.com/nf-core/metaboigniter/issues/80
https://github.com/nf-core/metaboigniter/issues/78

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metaboigniter/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metaboigniter _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by AinaVaivade — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v2.0.1](https://github.com/nf-core/metaboigniter/releases/tag/2.0.1) - 2024-04-12
+
+- Several bug fixes
```
Comment (maxulysse) [inline CHANGELOG.md:8]: Can you detail this part more?

**Review by maxulysse — COMMENTED** [review]
minor comments

File: CHANGELOG.md (hunk 1 above)
Comment (PayamEmami) [reply CHANGELOG.md]: Done!

**Comment by PayamEmami** [comment]
@maxulysse Thanks for the review. Any other comments?

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
> @maxulysse Thanks for the review. Any other comments?

No, all good to me

**Pull request merged**